string action_state
string robot_db
int16 block_id
